cd-cli/cnf-templates/complete-pipeline.yaml

```diff
@@ -5059,7 +5059,7 @@ Resources:
     Type: AWS::Events::Rule
     Properties:
       Description: "e2eScheduledRule"
-      ScheduleExpression: "cron(0 1 ? * MON-FRI *)"
+      ScheduleExpression: "cron(3 4 ? * MON-FRI *)"
```

Comment:
@leleOFA questa è la pipeline dei test di BE, modifica il cron alla riga 5070 della FeScheduledRule

Reply:
done